enterprise/ssl-impl/build.gradle

```diff
@@ -4,14 +4,23 @@ archivesBaseName = 'ssl-impl'
 group = 'io.crate'
 description = 'SSL Encryption Implementation for CrateDB'
 
+repositories {
+    maven {
+        url "https://repo.eclipse.org/content/repositories/paho-releases/"
+    }
+}
+
 dependencies {
     compile project(':ssl')
 
     testCompile project(':sql')
+    testCompile project(':enterprise:mqtt')
     testCompile project(':integration-testing')
     testCompile project(path: ':sql', configuration: 'testOutput')
     testCompile project(path: ':dex', configuration: 'testOutput')
-    testCompile('io.crate:crate-jdbc:2.1.5') {
+    testCompile project(path: ':enterprise:mqtt', configuration: 'testOutput')
+    testCompile('org.eclipse.paho:org.eclipse.paho.client.mqttv3:1.2.0')
+    testCompile('io.crate:crate-jdbc:2.1.7') {
```

Comment:
Shouldn't this work the other way around? The MQTT module should depend on the `ssl-impl` module.

Reply:
:+1: I'm also in favor of swapping the dependencies